main/core.js

```diff
@@ -64,9 +65,10 @@ async function start (ctx) {
 
   assert(childProcess.stdout)
   childProcess.stdout.setEncoding('utf8')
-  childProcess.stdout.on('data', chunk => {
-    logs.push(chunk)
-    for (const line of chunk.split('\n').filter(Boolean)) {
+  childProcess.stdout
+    .pipe(split2())
+    .on('data', line => {
+      logs.push(line)
       const event = JSON.parse(line)
```

Comment:
I think we should wrap this line in a try/catch block so that a non-JSON line does not crash the process. WDYT?